app/models/live_metric.rb

```diff
@@ -3,6 +3,10 @@ class LiveMetric < ActsAsArModel
 
   class LiveMetricError < RuntimeError; end
 
+  def self.virtual_attribute?(c)
+    super || c =~ /^mw_/
```

Comment:
Thanks @lucasponce 
I had seen that regular expression in a few places, so thought that was a good test.
I'll just return true there.